### ISSUE TITLE
MQTTS proxy not working

### ISSUE THREAD

**Issue by SaraVeterini** [NVISO-BE/IOXY#2]
When selecting **mqtts** proxy mode, after inserting certificates and applying changes the proxy blocks, and prints only the mitm broker certificates in the console.
The certificates are in .pem format. How can I configure correctly the certificates in the mitm and distant broker configuration?

Thank you.

**Comment by windBlaze** [comment]
Hi Sara, thank you for the report! I'll investigate over the next few days and come back to you!

**Comment by SaraVeterini** [comment]
Hi, I have an update. I tried selecting mqtts mode both in mitm broker and in distant broker settings, then I added the same **key-pem** set (mitm.key and mitm.pem files) in both settings, **without the root CA in MITM and the CA in distant broker**, and it worked. So the problems seems to be the CA management. The CA I used before is installed in my machine.

I also tried putting mqtt in mitm broker (so without certificates), and mqtts in distant broker, with the key-pem pair **and the ca.pem file**. In this way it works too, meaning that **the CA is a problem only when combined with MITM broker settings in MQTTS mode**.

I hope this helps in reproducing the issue. I think this tool can be very powerful.

Thank you.
